src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreUtils.cs

```diff
@@ -15,9 +15,8 @@ namespace Raven.Server.Documents.PeriodicBackup.Restore
     public static class RestoreUtils
     {
         private static readonly Regex BackupFolderRegex = new Regex(@".ravendb-(.+)-([A-Za-z]+)-(.+)$", RegexOptions.Compiled);
-        private const string LegacyIncrementalBackupExtension = "ravendb-incremental-dump";
-        private const string LegacyFullBackupExtension = "ravendb-full-dump";
-        
+        private static readonly Regex FileNameRegex = new Regex(@"([0-9]{4}-[0-9]{2}-[0-9]{2}-[0-9]{2}-[0-9]{2})", RegexOptions.Compiled);
```

Comment:
rename to LegacyFileNameRegex?

Reply:
this applies to our current backups as well (but should be very rare)

2018-01-01-13-01-1.ravendb-incremental-backup
2018-01-01-13-01-2.ravendb-incremental-backup

Reply:
ok